frontend/package-lock.json

```diff
@@ -133,6 +134,7 @@
         "@tooljet-plugins/typesense": "file:packages/typesense"
       },
       "devDependencies": {
+        "@types/nodemailer": "^6.4.4",
```

Comment:
Do we need nodemailer as devDependency in front end?

Reply:
No, it was added by accident

Reply:
Can you please fix it?